source/index.html.md

```diff
@@ -182,6 +182,9 @@ error categories, the last two digits define specific errors.
 ### Unsupported HTTP method (400 error)
 * 0901: Unsupported HTTP method
 
+### Withdrawals error codes
+* 1000: "Error processing the withdrawal"
```

Comment:
This error code is now used See. https://github.com/bitsoex/slate/pull/45

Please change it

Reply:
Fixed